.github/actions/repo-gardening/src/tasks/notify-design/index.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+const { getInput } = require( '@actions/core' );
+
+/**
+ * Internal dependencies
+ */
+const debug = require( '../../debug' );
+const getLabels = require( '../../get-labels' );
+const sendSlackMessage = require( '../../send-slack-message' );
+
+/* global GitHub, WebhookPayloadPullRequest */
+
+/**
+ * Check for a Design Review status label on a PR.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasDesignReviewLabel( octokit, owner, repo, number ) {
+	const labels = await getLabels( octokit, owner, repo, number );
+	// We're only interested in the Needs Design Review label.
+	return !! labels.find( label => label.match( /^\[Status\]\sNeeds\sDesign\sReview$/ ) );
```

Comment:
Why `\s` instead of just a space? Is there a chance the label returned is going to contain other kinds of spaces?

And in that case, you could simplify it even more to just
```suggestion
	return labels.includes( '[Status] Needs Design Review' );
```

Reply:
Good point, that's simpler.